src/site/stages/build/drupal/assets.js

```diff
@@ -29,10 +29,10 @@ function replacePathInData(data, replacer) {
   return current;
 }
 
-function convertAssetPath(drupalInstance, url) {
+function convertAssetPath(url) {
   // After this path are other folders in the image paths,
   // but it's hard to tell if we can strip them, so I'm leaving them alone
-  const withoutHost = url.replace(`${drupalInstance}/sites/default/files/`, '');
+  const withoutHost = url.replace(/^.*\/sites\/.*\/files\//, '');
```

Comment:
Rather than futzing with the string literal combined with a regex, it seemed that just a regex was all that was needed.

Reply:
Would it make sense to store this regex in a variable for reuse in `replacePathInData` (line 85)?

Will this affect `localhost` builds at all?

Reply:
I thought about that but since it was split over two functions I thought the readability was better with them separately. If you want it though, can you help me with scope on that. Been a while since I JSd.